Makefile.dune

```diff
@@ -55,7 +55,7 @@ states: voboot
 	dune build --display=short $(DUNEOPT) dev/shim/coqtop-prelude
 
 world: voboot
-	dune build $(DUNEOPT) @install
+	dune build $(DUNEOPT) coq.install coqide-server.install coqide.install
```

Comment:
Since these arguments are used in 2 places (the watch target and here) to replace `@install` maybe they should be factored into a variable `INSTALL_TARGETS:=coq.install coqide-server.install coqide.install`
Also, do `@install` and `coqide.install` do the same thing (fail/not fail) when the gtk libraries for coqide are missing or is this a behaviour change?

Reply:
I think so yes, `@install` is just an alias for all `.install` targets.

Reply:
We need to check tho as the ide rules include some optional and if rules.

Another way to go is define an alias `@world` for these packages, I'm unsure if that's equivalent. By the way we could try to make compilation of the documentation conditional.

Reply:
Hum. I tried to build without the CoqIDE dependencies (`nix-shell --arg buildIde false`) but actually `make -f Makefie.dune coqide` succeeds (because of my local Dune cache I suppose).

Reply:
It likely produces an almost empty package?

Reply:
My testing seems to confirm my hypothesis which was that the targets behave the same. In fact, none of these targets fail. They just build less when CoqIDE dependencies are not satisfied (I'm not convinced by this behavior BTW).

Reply:
That was done on purpose so people without ide libs don't get errors on @install but indeed I'm not sure it was the right choice.